ecl/eclagent/eclagent.cpp

```diff
@@ -2498,10 +2498,12 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
 
     if (!isResult(lfn, ResultSequencePersist))
         errText.appendf("Building PERSIST('%s'): It hasn't been calculated before", logicalName);
-    else if (!isResult(crcName, ResultSequencePersist))
-        errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
     else if (isFile && !fileExists(logicalName))
         errText.appendf("Rebuilding PERSIST('%s'): Persistent file does not exist", logicalName);
+    else if (!item.queryPersistRefresh())
+        return true;
```

Comment:
I think we should get some tracing output here if continuing past a REFRESH(false), e.g.
Not rebuilding PERSIST('%s'): due to REFRESH(false)